app/views/layouts/angular-bootstrap/_endpoints_angular.html.haml

```diff
@@ -77,6 +78,17 @@
                           "checkchange"             => "",
                           "prefix"                  => prefix.to_s,
                           "reset-validation-status" => "#{prefix}_auth_status"}
+    - detect = "detectClicked({target: '.detect_button'})"
+    %span
+      %miq-button.detect_button{"ng-if" => detection ? 'true' : 'false',
+                  :name            => _("Detect"),
+                  :on_click        => detect,
```

Comment:
@himdel these attribute names seem to work.
Do you want them changed to strings with -?
(:on_click, :disabled_title, :enabled_title:)